hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/DelegateCallOperationSuite.java

```diff
@@ -71,13 +74,38 @@ HapiSpec verifiesExistence() {
                             final var solidityAddress = HapiPropertySource.asHexedSolidityAddress(id);
 
                             final var contractCall = contractCall(
-                                            contract, "delegateCall", asHeadlongAddress(solidityAddress))
+                                    contract, "delegateCall", asHeadlongAddress(solidityAddress))
                                     .hasKnownStatus(SUCCESS);
 
                             allRunFor(spec, contractCall);
                         }));
     }
 
+    @HapiTest
+    HapiSpec testCallOperationsForSystemAccounts() {
+        final var contract = "CallOperationsChecker";
+        final HapiSpecOperation[] opsArray = new HapiSpecOperation[systemAccounts.size() * 4];
```

Comment:
Add cases where you send some value to the payable functions, if you haven't already.

Reply:
Also, add check if the call is successful in the contract, like this:
```solidity
        (bool sent, bytes memory data) = _to.call{value: msg.value}("");
        require(sent, "Failed to send Ether");
```

Reply:
Changed the tests so that they verify if internal call is success